chapters/beam_loader.asciidoc

```diff
@@ -49,3 +49,231 @@ e.g. +erts/emulator/x86_64-unknown-linux-gnu/opt/smp/+).
 
 The same program (beam_makeops) also generates the Erlang code for the
 compiler back end +beam_opcodes.erl+.
+
+=== Understanding ops.tab
+
+// Missing description about build specific options, i.e. the %unless construct
```

Comment:
I was referring to this kind of admonitions:

http://asciidoctor.org/docs/user-manual/#admonition

For now let's park the discussion until the guidelines are in place. Comment is fine.